.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
```

Comment:
Kindly set up your linters according to[ this file](https://github.com/microverseinc/linters-config/tree/master/html-css):
Just follow the steps on the page, you have some missing files (hintrc, styleinterc.json..)